flw/src/main/java/org/motechproject/nms/flw/service/impl/FrontLineWorkerImportServiceImpl.java

```diff
@@ -182,6 +183,7 @@ private String readLineWhileNotBlank(BufferedReader bufferedReader) throws IOExc
         mapping.put(FlwConstants.VILLAGE_NAME, new Optional(new GetString()));
 
         mapping.put(FlwConstants.TYPE, new Optional(new GetString()));
+        mapping.put(FlwConstants.UPDATED_ON, new Optional(new GetLocalDate()));
```

Comment:
Csv Import which are blocked from mcts retrial are fixed here.So added Updated_On field for mapping and respective testcases.